libs/shared/utils/src/getPaths/isSlugRestricted.ts

```diff
@@ -1,7 +1,7 @@
 import { restrictedSlugs } from './restrictedSlugs';
```

Comment:
Maybe this is where the fix needs to happen?

'post' doesn't appear in `restrictedSlugs` -- it [instead (still?) has `blog`](https://github.com/Quansight/Quansight-website/blob/86078fe8516aa9e3dfa863a5479cc000c475712d/libs/shared/utils/src/getPaths/restrictedSlugs.ts#L10), which I think was relevant before we located the blog posts to `/post`, and is out of date now.

Reply:
- https://github.com/Quansight/Quansight-website/pull/663/files

Reply:
:+1: to removing `'blog'` ... Should we also add `'^/post/`?

That may fix the post double-rendering.

Or, it may entirely break the blog build?

Reply:
Ahhhh. Nevermind, I made it to the end of the PR thread. Got it.